### PR TITLE
Write json file, don't track h5 file

### PR DESCRIPTION
**Pull request by Edgar-21**
I added a line to write the json representation of the library, this at least allows for tracking of the output of the material scripts to ensure they don't change when we don't expect them to. I also removed the h5 version of the library from tracking.

Added quotes to a string.

I also wrote the material library from the main branch to json to compare to the one made wit this branch and found zero changes, which is good to see.

Wasn't sure if I should push directly to the branch or PR to it, this felt like the better option.

### CODE REVIEW

**Comment by MicahGale** [comment]
You may want to update `.gitignore` as well.

**Comment by Edgar-21** [comment]
good call, thanks

**Comment by Edgar-21** [comment]
should we include any other formats or just the JSON?

File: pureMaterials/createPurematlib.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def make_mat_from_atom(atom_frac, density, citation):
 # added 10 wppm B which is important to assess typical He production level
 mat_data['SS316L'] = {
     'nucvec' : {50000000:0.001, 60000000:0.03, 140000000:1.0, 150000000:0.045, 160000000:0.03, 240000000:17, 250000000:2, 260000000:65.394, 280000000:12, 420000000:2.5},
-    density : 8.00,
+    'density' : 8.00,
```
Comment (gonuke) [inline pureMaterials/createPurematlib.py:104]: Nice catch

**Review by gonuke — APPROVED** [review]
Thanks for this tweak @Edgar-21

**Comment by Edgar-21** [comment]
I think this is because there was a commit pushed to #13 after I made this PR, I could rebase this branch to include the latest commit on #13? Or make a PR reverting the latest commit on #13, though I'm not 100% sure that that would do the trick. 

Github is suggesting the following as well:

```
git checkout -b Edgar-21-write_json streamline-create-pure
git pull git@github.com:Edgar-21/fusion-material-db.git write_json

git checkout streamline-create-pure
git merge --no-ff Edgar-21-write_json
git push origin streamline-create-pure
```
But this involves pushing straight to streamline-create-pure, I suppose I could PR the merged result as well.

**Comment by gonuke** [comment]
Nope - don't do that... we have a little git confusion we need to clear up

**Comment by Edgar-21** [comment]
looks like the revert did the trick

**Review by gonuke — APPROVED** [review]

**Pull request merged**